src/lib/daterangepicker.less

```diff
@@ -9,6 +9,10 @@
 /* Nodier 
 changed left and right rule names to prevent clash with sense client css */
 
+#grid .qv-gridcell.zoom, .sheet-list #grid .qv-gridcell.zoom {
```

Comment:
Does this need to apply to all gridcells that are zoomed or could it be more specific to only apply to datepickers, adding `.qv-object-qlik-date-picker` to the selector?

Reply:
I tried to do it locally in date-picker, but #grid is the parent to all the objects opening in Single Object View or Maximize mode and the z-index was overridden. With this solution this rule only applies to the maximized or an object opening in single object view as for the specific class .qv-gridcell.zoom. And that object can have the highest z-index.